README.md

```diff
@@ -1,18 +1,18 @@
 ![The mrjs logo](https://docs.mrjs.io/static/mrjs_nopadding.svg)
  
-An extensible WebComponents library for the Spatial Web
+An extensible Web Components library for the Spatial Web
```

Comment:
Components should be singular (--> Component)

Reply:
🤔 or maybe not - it sounds wrong as plural though, but looking again maybe?

i'm just going to leave this as highlighted and let you both determine the final version you want for this one

Reply:
"An extensible library of Web Components for the spatial web"?

Reply:
yea that sounds better